hyperspy/roi.py

```diff
@@ -209,13 +209,14 @@ def _parse_axes(self, axes, axes_manager):
 
             * a tuple of:
 
-              * :py:class:`~hyperspy.axes.DataAxis`. These will not be checked
-                with signal.axes_manager.
-              * anything that will index the signal
-                :py:class:`~hyperspy.axes.AxesManager`
-            * For any other value, it will check whether the navigation
-              space can fit the right number of axis, and use that if it
-              fits. If not, it will try the signal space.
+                - DataAxis. These will not be checked with signal.axes_manager.
+                - anything that will index signal.axes_manager
+            * Anything that can index signal.axes_manager
```

Comment:
Maybe have this entry before the 'a tuple of'. Otherwise, it is weird that it says the same as in the line above. Why once 'will index' and once 'can index'?